README.md

```diff
@@ -1,113 +1,133 @@
 
-three files you need to focus on, 
+We define all the test cases under [the testcase directory](https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/testcases), and each sub folder will be treated as a test case. 
 
-* [go module to add your component](https://github.com/aws-observability/aws-otel-collector/blob/main/go.mod)
-* [enable your component](https://github.com/aws-observability/aws-otel-collector/blob/main/pkg/defaultcomponents/defaults.go).
-* [link your testcase](https://github.com/aws-observability/aws-otel-collector/blob/main/e2etest/testcases.json)
+You will need to create a sub folder under [the testcase directory](https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/testcases), and place your configuration into it. Typically, 
+you will need to place files as following(please use the same filename as below):
 
-## 2. Run and debug your testcase
+1. `otconfig.tpl`: which contain the new component, will be used as the config in all types of the e2etest. 
+2. [optional] `parameters.tfvars`: you can run your test case locally with adding `-var-file=../testcases/yourtestcase/parameters.tfvars` to the terraform command. By using this file, you can override the default configuration in the testing framework. [The parameters you can override](terraform/README.md).
+4. [optional] `docker_compose.tpl`: which is used to launch sample app container in ec2 test.
+5. [optional] `ecs_taskdef.tpl`: which is used to launch ecs task in ecs test.
+6. [optional] `eks_pod_config.tpl`: which is used to launch eks pod in eks test.
 
-You can run the testcase on your local. follow the below guide to set up testing framework on your local.
+all the default files can be found [here](https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/templates/defaults)
 
-### 2.1 Prerequisite
+### 2.2 Link test case
 
-#### 2.1.1 Setup your aws credentials
+In the PR you create in [AWS Otel Collector](https://github.com/aws-observability/aws-otel-collector) to build the new component,
+[link your testcase](https://github.com/aws-observability/aws-otel-collector/blob/main/e2etest/testcases.json)
 
-please check https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+When you link your testcase, there are five types of testing platforms you can choose,
+* LOCAL, which will run the test in the pr workflow. 
+* EC2, which will run the test in the main branch workflow under an ec2 instance.
+* ECS, which will run the test in the main branch workflow under an ecs cluster.
+* EKS, which will run the test in the main branch workflow under an eks/k8s cluster.
+* SOAKING, which will run every night, and perform high throughput to your test case and monitor the resource usages.
 
-#### 2.1.2 install terraform
+Typically, we required all the types for every test case, if you have a special usage of your component which might not need a certain platform,
```

Comment:
a little confusing phrase. can we say "we require the new components to be tested on all platforms for all the test cases."